psi4/src/psi4/libmints/extern.cc

```diff
@@ -92,6 +92,25 @@ void ExternalPotential::print(std::string out) const {
     }
 }
 
+std::string ExternalPotential::get_xyz() {
+
+    std::string xyz = "";
+    size_t n_atoms = 0;
```

Comment:
Why do you even have this variable? It sounds like you just want `charges_.size()` in `std::to_string`.

Reply:
Fixed.